### PR TITLE
retrospective blog post

### PR DESCRIPTION
**Pull request by bketelsen**

### CODE REVIEW

File: content/gophercon-2016/gophercon2016-retrospective.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+author = ["Brian Ketelsen", "Erik St. Martin"]
+date = "2016-07-20"
+title = "GopherCon 2016 - Retrospective"
+tags = ["gophercon","community"]
+series = ["GopherCon 2016"]
+
++++
+
+GopherCon 2016 has come and gone, and in keeping with our tradition we wanted to write up a brief retrospective to capture some of the highlights of the event.  It’s hard to believe it happened so quickly --- we spend months planning all the details and then when the conference arrives it is over in the blink of an eye.  This year we were joined by 1400 Gophers from 38 different countries.  Yes… 38 different countries.  The diversity of people and thoughts at GopherCon was amazing, thanks in part to everyone who donated to the Diversity Scholarship.  We’d like to thank [VividCortex](https://www.vividcortex.com/) especially for their corporate donation.  A community conference should represent the whole community, and our goal is to make the sea of faces at GopherCon look like the faces of Gophers everywhere.  Thanks to everyone who helped get us closer to this goal by purchasing the adorable hoodies.  Together, we helped 37 people come to the conference this year, and that’s no small accomplishment!
```
Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: perhaps mention that the conference matched each donation dollar for dollar ?

Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: s/come to/attend/ and maybe break it down

"helped 37 people with flights, hotel, or entry tickets where appropriate so they could attend the conference"

File: content/gophercon-2016/gophercon2016-retrospective.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Another first for this year was real-time captioning sponsored by our friends at [Digital Ocean](http://www.digitalocean.com).  We heard from several people that the captioning made it easier for everyone to enjoy the conference and understand the speaker, so we were pleasantly surprised that the benefit extended beyond those with hearing disabilities. Many of the people who were enjoying the live stream commented on the live captioning too.  It made the entire event more accessible and enjoyable for everyone.  The real-time transcription service was provided by [White Coat Captioning](http://whitecoatcaptioning.com) and we just don’t have enough words to express our gratitude for their service.  Our captioner Stacey Potenza was lightning quick and had a great technical background.  Here’s [her take on the event](https://storify.com/whitecoatcapxg/gophercon-2016-57862d7dbcf2a9665c234aa7). We were all impressed at how effortlessly she transcribed even the most technical of talks.  You’ll definitely see this next year at GopherCon, and we hope this catches on at other conferences.
```
Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: link to Stacey's twitter handle ?

File: content/gophercon-2016/gophercon2016-retrospective.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+A new addition this year was the Guide Program, organized by [Andy Walker](http://twitter.com/alaskacodes).  Andy did a terrific job of pairing seasoned conference veterans with people who were looking for a guide.  Our Guides made sure that their counterparts knew where to be and gave valuable advice to those who participated.  The Guide Program was a big hit, with several first-time conference attendees getting more benefit out of the conference because of the assistance and advice the Guides gave them.  We hope that these relationships continue beyond the conference, and we’re grateful to everyone who participated.  Thank you Andy for putting this together and making it all work!  One thing that we heard over and over last week was that the feeling of inclusion in the Go community is strong.  We hope the Guide program helps keep the sense of community strong and sets the tone for the people who are new to the community.
```
Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: s/relationships/friendships ?

File: content/gophercon-2016/gophercon2016-retrospective.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Our third day, or Hack Day, was even bigger this year.  Our Hack Day attendance doubled this year, which caused us some last minute scrambling in the catering department.  We also noticed that many had trouble finding like-minded Gophers to hack with.  We quickly added white boards where folks could sign up with a table number so it was easier to find your friends and others interested in working on projects together.  Another big hit on Hack Day was the “Go Project Room”.  We recognize the importance of the community being able to communicate with the Go Team directly in this format, so we’ll expand that room next year and provide more space and Audio/Visual equipment to foster this valuable communication.  The Hack Day will likely be renamed to something more fitting next year -- perhaps “Community Day”.  It was great to see everyone collaborating, communicating and having fun together in a loosely organized format like this.
```
Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: s/we/Heather or Conference designs.

File: content/gophercon-2016/gophercon2016-retrospective.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+|   | [Digital Ocean](http://www.digitalocean.com) &nbsp;| [Pivotal](http://www.pivotal.io)  |  [Heroku](http://www.heroku.com) | [LinkedIn](http://www.linkedin.com)  |   |
+|   |   | [New Relic](http://www.newrelic.com)  | [YikYak](http://www.yikyakapp.com)  | [Netflix](http://www.netflix.com)  |   |
+|   |   |   |  [backplane](http://backplane.io) | [RedVentures](http://www.redventures.com) &nbsp; |   |
+|   | |   |   | [Convox](http://convox.com)  |   |
+|   |  |   |   |  |   |
+
+
+At the end of the day, a conference will be judged by the experience at the event and by the speakers it puts on the stage.  We think this year’s speakers were the best we’ve ever had.  Thank you to [Dave Cheney](http://www.twitter.com/davecheney) for curating our program this year and making sure that first-time speakers had a mentor to help them bring their best talk to the big stage.  We were blown away by the professionalism and preparation of the speakers this year.  You can see the slides [here](http://github.com/gophercon/2016-talks), and we’ll update that repository with links to the videos when they’re ready.
```
Comment (davecheney) [inline content/gophercon-2016/gophercon2016-retrospective.md]: Can you give a rough date when they might be ready, even if its vague; ie "by the end of next month". or something.

**Comment by davecheney** [comment]
👍 ship it!

**Comment by bketelsen** [comment]
add gobot / Ron

File: content/gophercon-2016/gophercon2016-retrospective.md (hunk 1 above)
Comment (bketelsen) [inline content/gophercon-2016/gophercon2016-retrospective.md]: 36 countries says E

File: content/gophercon-2016/gophercon2016-retrospective.md (hunk 2 above)
Comment (bketelsen) [inline content/gophercon-2016/gophercon2016-retrospective.md]: Originally budgeted and booked for mornings, we begged her to stay all day.

**Pull request merged**